libs/test-commons/src/libs/test-commons.ts

```diff
@@ -9,3 +12,10 @@ export const shouldRunOnlyOn = (impl: string[], current: string) => impl.include
 // eslint-disable-next-line @typescript-eslint/ban-ts-comment
 //@ts-ignore
 export const testIfSupportedOperationsIncludes = (supportedOperations: string[], operation: string[]): any => operation.every((o: any) => supportedOperations.includes(o)) ? test : test.skip 
```

Comment:
I know tgat you didn't change this code, but maybe on the way change the supportedOperations, operation to SchemaOperations type

Reply:
And if you have a way to remove the ts-ignore from there it would be great as well  😅